src/plugins/cal-events/cal-events.js

```diff
@@ -256,6 +259,15 @@ var componentName = "wb-calevt",
 		// End of loop through objects/events
 		}
 
+		// Set the min and max Date according to the current date.
+		if ( events.minDate.getTime() > currDate.getTime() ) {
+			events.minDate = currDate;
```

Comment:
Same issue as noted noted previously https://github.com/wet-boew/wet-boew/pull/7671#discussion_r86915152

It's still looks like this already being done in the loop before: https://github.com/jilusan/wet-boew/blob/e772fc680be81434b25dade9f19945fbbd31d6a0/src/plugins/cal-events/cal-events.js#L226